src/jdk.incubator.code/share/classes/jdk/incubator/code/op/CoreOp.java

```diff
@@ -4435,4 +4435,84 @@ public static BinaryTestOp le(Value lhs, Value rhs) {
     public static ConcatOp concat(Value lhs, Value rhs) {
         return new ConcatOp(lhs, rhs);
     }
+
+    public static FuncOp quoteOp(Op op) {
```

Comment:
See also the method `BytecodeGenerator::quote` it does the same thing specific to `LambdaOp`, and we can generalize to `Op` and then the bytecode generator can reuse the general one.